app/src/main/java/com/orange/ouds/app/ui/about/AboutFileScreen.kt

```diff
@@ -37,8 +36,8 @@ private const val FilePath = "file:///android_res/$FileResourceDir/"
 @Composable
 internal fun AboutFileScreen(fileMenuItem: AboutFileMenuItem) {
     val context = LocalContext.current
-    val horizontalPadding = dimensionResource(id = R.dimen.screen_horizontal_margin).value
-    val verticalPadding = dimensionResource(id = R.dimen.screen_vertical_margin).value
+    val horizontalPadding = OudsTheme.grids.margin.value
```

Comment:
On my device the margin is now 24dp instead of 16dp. Are we sure that 24dp is correct or should we use another token?

Reply:
I think it is correct as you can see here: [https://unified-design-system.orange.com/472794e18/p/26a48e-grid/b/72c7e2/t/1038e734a0](https://unified-design-system.orange.com/472794e18/p/26a48e-grid/b/72c7e2/t/1038e734a0)